src/application/models/v3/Appointments_model_v3.php

```diff
@@ -104,10 +104,17 @@ protected function get_aggregates(array $appointment)
     }
 
     public function getAppointmentsWorkingDate($id_service_integrated, $id_provider_integrated, $dates) {
-        $sqlQuery = "SELECT DATE(ea_appointments.start_datetime) as date from ea_appointments 
-                    INNER JOIN ea_users ON ea_appointments.id_users_provider = ea_users.id 
+        $sqlQuery = "SELECT ea_customer.email, 
+                            ea_appointments.id_integrated as bookingId, 
+                            CONCAT(ea_provider.first_name,' ',ea_provider.last_name) as doctorName,
```

Comment:
dont introduce new model if it is not needed with new fields { bookingId, doctorName, customerName, date, time }

instead can you change to reuse the current model, the appointment

Reply:
updated